src/execution/sql/ddl_executors.cpp

```diff
@@ -156,4 +156,9 @@ bool DDLExecutors::CreateIndex(const common::ManagedPointer<catalog::CatalogAcce
   TERRIER_ASSERT(result, "CreateIndex succeeded, SetIndexPointer must also succeed.");
   return true;
 }
+
+bool DDLExecutors::CreateViewExecutor(common::ManagedPointer<planner::CreateViewPlanNode> node,
+                                      common::ManagedPointer<catalog::CatalogAccessor> accessor) {
+  return true;
```

Comment:
Is the executor also targetting a future implementation of CREATE VIEW? If not, don't we need any ASSERT statements before returning?